MediaBrowser.XbmcMetadata/Savers/EpisodeNfoSaver.cs

```diff
@@ -109,7 +108,8 @@ protected override List<string> GetTagsUsed(BaseItem item)
             return list;
         }
 
-        public EpisodeNfoSaver(IFileSystem fileSystem, IServerConfigurationManager configurationManager, ILibraryManager libraryManager, IUserManager userManager, IUserDataManager userDataManager, ILogger logger, IXmlReaderSettingsFactory xmlReaderSettingsFactory) : base(fileSystem, configurationManager, libraryManager, userManager, userDataManager, logger, xmlReaderSettingsFactory)
+        public EpisodeNfoSaver(IFileSystem fileSystem, IServerConfigurationManager configurationManager, ILibraryManager libraryManager, IUserManager userManager, IUserDataManager userDataManager, ILogger logger)
```

Comment:
drop `fileSystem`?

Reply:
Can't the base class needs it atm.

Reply:
Cannot it be dropped from base class as well?